### DISCUSSION TITLE
api /sb/targets whether can support expressions type asinAgeRangeSameAs and asinIsPrimeShippingEligible？

### DISCUSSION THREAD

**Discussion by LionSniper** [amzn/ads-advanced-tools-docs#251] [Q&A] [answered]
i want create sb target，but i find in the docs that sb target create api(/sb/targets) only support expressions type include "asinCategorySameAs" "asinBrandSameAs" "asinPriceLessThan" "asinPriceBetween" "asinPriceGreaterThan" "asinReviewRatingLessThan" "asinReviewRatingBetween" "asinReviewRatingGreaterThan" "asinSameAs", i want ask question, sb target expression type whether can support  "asinAgeRangeSameAs" and "asinIsPrimeShippingEligible"? if it can support, please tell me how to use it,thanks!
@elizafarley

**Comment by elizafarley** [comment, marked as answer]
SB targets currently only support the refinements you listed: "asinCategorySameAs" "asinBrandSameAs" "asinPriceLessThan" "asinPriceBetween" "asinPriceGreaterThan" "asinReviewRatingLessThan" "asinReviewRatingBetween" "asinReviewRatingGreaterThan" "asinSameAs". Only SD and SP support "asinAgeRangeSameAs" and "asinIsPrimeShippingEligible" for now. Hope that helps!
